<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CManualConfirmDialog</class>
 <widget class="QDialog" name="CManualConfirmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="le_verify_code">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>94</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Cancel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>217</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_remark">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>144</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Title">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>审核驳回</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_remark">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>145</y>
      <width>54</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>备注：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OK">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>217</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_verify_code">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>95</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>谷歌验证码：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
